taxi/admin.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ("license_number", )
+    fieldsets = UserAdmin.fieldsets + (
+        ("Additional info", {"fields": ("license_number", )}),
+    )
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        ("Additional info", {"fields": ("license_number", )}),
+    )
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_filter = ["model", "manufacturer"]
```

Comment:
To fulfill the requirement of making it possible to search `Car` by `model`, you should add `search_fields = ['model']` to the `CarAdmin` class.